From 1abe9f606b93e43c2c3110fb621832e8a42abe11 Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Thu, 11 Jul 2013 12:49:46 +0800
Subject: [PATCH 212/602] ENGR00215174 add WfdSink and fsl.imx.jar to system
 image.

add WfdSink and fsl.imx.jar to system image.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 imx6/imx6.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index f6c4617..2fdb7ce 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -19,6 +19,9 @@ PRODUCT_PACKAGES += \
 	Email					\
 	FSLOta					\
 	CactusPlayer                            \
+	WfdSink                                 \
+	libfsl_wfd.so                           \
+	fsl.imx.jar                             \
 	VideoEditor				\
 	FSLProfileApp				\
 	FSLProfileService			\
-- 
1.8.0

